src/ralph/models/xapi/config.py

```diff
@@ -9,3 +9,11 @@ class BaseModelWithConfig(BaseModel):
     class Config:  # pylint: disable=missing-class-docstring # noqa: D106
         extra = Extra.forbid
         min_anystr_length = 1
+
+
+class BaseExtensionModelWithConfig(BaseModel):
+    """Pydantic model for extension configuration shared among all models."""
+
+    class Config:  # pylint: disable=missing-class-docstring # noqa: D106
+        extra = Extra.allow
+        min_anystr_length = 1
```

Comment:
Aren't empty strings allowed in extension fields? Maybe we could use the `BaseModel` directly? (Extra.allow is used by default) WDYT?

Reply:
You're right, empty strings should be allowed for all extension fields, even for `object.definition.extensions`. 

Default extra seems to be `ignore` instead of `allow`. It should be best to allow extra extensions as the `validate` command returns a JSON string of the model parsed/validated.